<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>balance</class>
 <widget class="QWidget" name="balance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="INIlistWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>711</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="moneyAll_lb">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>534</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up_bt">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交订单</string>
   </property>
  </widget>
  <widget class="QLabel" name="moneytext_lb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>总价：¥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_bt">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消订单</string>
   </property>
  </widget>
  <widget class="QLabel" name="fail_lb">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>580</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>余额不足！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
